spec/models/participant_spec.rb

```diff
@@ -20,4 +20,27 @@
       end
     end
   end
+  describe 'total score' do
+    it 'should be zero' do
+      expect(participant.total_score).to eql 0
+    end
+    it 'should increase by 40 from quiz score' do
```

Comment:
I think what this is actually testing is that adding a quiz score updates the total score. It'd be better if the test didn't depend on specific numbers, so something more like:

```ruby
quiz_score = create :quiz_score
expect { participant.update quiz_score: quiz_score }
  .to change { participant.total_score }
  .by(quiz_score.score)
```